### ISSUE TITLE
hassio release (2022.2.3?)

### ISSUE THREAD

**Issue by ccdunder** [sbidy/wiz_light#236]
First of all, I'm so grateful for all your work on this, and congrats on getting into hassio core!

I just wanted to double-check if it's made it into a hassio release. I see it in [`dev`](https://github.com/home-assistant/core/tree/dev/homeassistant/components/wiz) but not [`2022.2.8`](https://github.com/home-assistant/core/tree/2022.2.8/homeassistant/components/wiz). README.md mentions `look for the next upcoming release (>2022.2.3)` so I wonder if I'm missing something. Let me know if there's anything to do to help.

Thanks again,
-- Chris

**Comment by bdraco** [comment]
https://github.com/sbidy/wiz_light/discussions/78#discussioncomment-2177773

**Comment by ccdunder** [comment]
Thank you very much! 🙏

**tl;dr**
2022.3 beta = Feb 23)
2022.3 release = March 2)

> It will be available in the 2022.3 beta https://calendar.google.com/calendar/u/0/r/day/2022/2/23?eid=NGRvOGNodGE0ZWg2cnM3M2pqMnUxdDZsOWdfMjAyMjAyMjMgcDA3bjk4Z28xMW9uYW1kMDhkMGttcTZqaHNAZw&ctz=America/Los_Angeles&sf=true
> 
> And the 2022.3 release
> https://calendar.google.com/calendar/u/0/r/day/2022/3/2?eid=NWI0bnM3cGM1bG9qdHUzYTY0M3RzOGE5cHZfMjAyMjAzMDIgcDA3bjk4Z28xMW9uYW1kMDhkMGttcTZqaHNAZw&ctz=America/Los_Angeles&sf=true
